### DISCUSSION TITLE
TP1-Ejercicio2

### DISCUSSION THREAD

**Discussion by Clirussa** [INGCOM-UNRN-P1/cursada-2023#73] [Preguntas y respuestas] [answered]
Hola, nos estamos rompiendo la cabeza con @agosgit pero no logramos entender la sintaxis para que se tome el valor de temperatura y lo use el programa. Adjunto imagen de lo que fuimos pensando para armar el bloque de sentencias de la función. 

![ejercicio2](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/102642611/7abcb8bc-85c2-47ec-9948-028c2a3d2f9a)

**Comment by martinvilu** [comment, marked as answer]
¿Quien es `F`?, la variable no está definida.

recordá que a diferencia de Python, tenes que declarar la variable con su tipo

En ese código, está faltando algo como
```C
float F;
```
(Ojo con los nombres de variables, que sean descriptivos sin ser brütalmente largos)
Por lo que debiera ser
```C
float fahrenheit;
```

**Comment by BrunoCeballos** [comment]
Acordate que las variables dentro de las funciones son copias de lo que le ingresas a los argumentos. Al hacer "`centigrados = (5/9) * (F -32);`" estás intentando cambiar el valor "centigrados" que le ingresaste a la función, nomás (digo intentando porque el valor F no está declarado en la función), después igual estás retornando un 0 siempre.
Convendría obtener el valor de la conversión, almacenarla en un variable `float` y retornar esa variable.

**Comment by Clirussa** [comment]
Graciaaas!!
